narwhal/primary/src/primary.rs

```diff
@@ -674,7 +674,7 @@ impl WorkerToPrimary for WorkerReceiverHandler {
     ) -> Result<anemo::Response<()>, anemo::rpc::Status> {
         let message = request.into_body();
         self.tx_our_digests
-            .send((message.digest, message.worker_id))
+            .send((message.digest, message.worker_id, message.timestamp))
```

Comment:
Rather than send the three fields of the struct individually, at this point would it make more sense to just send the whole struct in the channel?

Reply:
The reason why I avoid doing that was to avoid leaking the network message outside of the "networking" related code/handler. But I agree that this tuple now starts becoming quite convoluted